openid-4-verifiable-presentations-1_0.md

```diff
@@ -687,6 +771,11 @@ This document also defines the following additional error codes and error descri
 
 - The Presentation Definition URL can be reached, but the specified `presentation_definition` cannot be found at the URL.
 
+`invalid_request_uri_mode`:
+
+- The value of the `request_uri_method` request parameter is neither `get` nor `post` (case-sensitive).
```

Comment:
Shouldn't that be "case-insensitive"?

Reply:
the definition of the request parameters defines it as case sensitive. we can change that, if you want

Reply:
But the definition says:

> Two case-insensitive values are defined for `request_uri_method` in this specification: `get` and `post`.


I'd prefer case-sensitive.